.gitmodules

```diff
@@ -16,3 +16,9 @@
 [submodule "external/Catch2"]
 	path = external/Catch2
 	url = https://github.com/catchorg/Catch2.git
+[submodule "external/parcon"]
```

Comment:
Is bringing parcon back intentional? If I recall correctly there were some licensing issues with this, which is why I replaced it with a homegrown BPF assember.

Reply:
Thanks for spotting this! It was a rebase error on my part